package.json

```diff
@@ -12,7 +12,7 @@
     "scriptPreprocessor": "<rootDir>/node_modules/babel-jest",
     "unmockedModulePathPatterns": [
       "<rootDir>/node_modules/react",
-      "<rootDir>/node_modules/react",
+      "<rootDir>/node_modules/fbjs",
```

Comment:
I'm not using fbjs so it's not needed here, but I duplicated:
```
"<rootDir>/node_modules/react",
```

so if you can update your PR with just removing that line I can merge it